test/paratii.eth.wallet.js

```diff
@@ -167,4 +167,22 @@ describe('paratii.eth.wallet: :', function () {
     })
     assert.equal(paratii.eth.wallet.getMnemonic(), undefined)
   })
+  it('eth.wallet.create() respect bip39 test vector', async function () {
+    var fs = require('fs')
+    var vector = JSON.parse(fs.readFileSync('test/testData/bip39-test-vector.json', 'utf8'))
```

Comment:
for JSON data we can use also just the require instead of fs, maybe is more lighter!